docs/tutorials/analyzing-ted-talks.md

```diff
@@ -43,22 +43,22 @@ talk and rating nodes.
 
 ## Exploring the dataset
 
-You have two options for exploring this dataset. 
-If you just want to take a look at the dataset and try out a few queries, open 
-[Memgraph Playground](https://playground.memgraph.com/) and continue with 
+You have two options for exploring this dataset.
+If you just want to take a look at the dataset and try out a few queries, open
+[Memgraph Playground](https://playground.memgraph.com/) and continue with
```

Comment:
Space at the end of the line doesn't create a new line as we previously discussed, you can check with the current live documentation, so I think it's safe to remove all of them from the files.